src/main/java/frc/robot/RobotContainer.java

```diff
@@ -30,19 +31,25 @@ public class RobotContainer {
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
 
+  private static BatterySubsystem batterySubsystem;
+
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
     if(Robot.isSimulation()){
       simulationSubsystem = new SimulationSubsystem(swerveSubsystem);
     }
-
+    if (!Robot.isCompetition) {
+      batterySubsystem = new BatterySubsystem();
+    }
     swerveSubsystem.setDefaultCommand(new SwerveTeleopCommand(
       swerveSubsystem, 
       OI.Driver.getXTranslationSupplier(),
       OI.Driver.getYTranslationSupplier(),
       OI.Driver.getRotationSupplier()));
 
-      
+    //Implement when SmartDashboard/Whatever system we use is ready, make sure to bind it to isCompetition in Robot.java
+    //SmartDashboard.putData(batterySubsystem);
+
```

Comment:
Is there a reason we can't put this in with the subsystem instantiation already?

Reply:
done, just removed it because we already use shuffleboard